README.md

```diff
@@ -39,18 +40,27 @@ Maybe it doesn't help that the first option explained in `man tar` is:
    and usually not even then as the default block size of 20 records (10240 bytes) is very common.
 ```
 
-Surely people could benefit from simplified man pages
-focused on practical examples.
+There seems to be room for simpler help pages, focused on practical examples.
```

Comment:
Totally a light suggestion, but I kinda liked the previous phrasing. Especially the "Surely people could benefit from.." part.

Reply:
Just to provide some context: I removed the "Surely..." to avoid making the phrasing of this section too repetitive, since the previous sentence was changed to start with "It certainly..."

```diff
-Maybe it doesn't help that the first option explained in `man tar` is:
+It certainly doesn't help that the first option explained in `man tar` is:
```

... which in turn was motivated by the inclusion of "Maybe" in

```diff
-New to the command-line world? Or just a little rusty?
+Maybe you are new to the command-line world? Or just a little rusty?
```

... which was written that way to make it flow better with the introductory paragraph I added to the top of the section:

> The **tldr-pages** project is a collection of community-maintained help pages
for command-line tools, that aims to be a simpler, more approachable complement
to traditional [man pages](https://en.wikipedia.org/wiki/Man_page).

So we can change this line back to what it was, but the preceding paragraphs would need to be adapted as well to avoid making the whole thing awkward or repetitive. Happy to hear any suggestions :)

Reply:
IMO, it doesn't look awkward to me. But I am totally unopinionated on this. It is a minor thing and I will defer to you.

Reply:
I appreciate your understanding. My preference is to either keep the set of changes I mentioned above, or adjust them all with alternative phrasings that would avoid the issues I pointed out above. Let me know what you guys think.

Reply:
Sounds good. No objections from me. Thanks for your work Waldir.